rollup/utils.js

```diff
@@ -19,7 +19,7 @@ export function mergePlugins(pluginsA, pluginsB) {
     // Hacky way to make sure the ones we care about end up where they belong in the order. (Really the TS and sucrase
     // plugins are tied - both should come first - but they're mutually exclusive, so they can come in arbitrary order
     // here.)
-    const order = ['typescript', 'sucrase', '...', 'terser', 'license'];
+    const order = ['excludeReplay', 'typescript', 'sucrase', '...', 'terser', 'license'];
```

Comment:
l: Wondering if it is worth adding a comment as to why it is important that this comes first - considering we discussed this when looking at this configuration.

Reply:
[10ff1a7](https://github.com/getsentry/sentry-javascript/pull/6508/commits/10ff1a7cd72592503ee65eef2c71f1eb0b1b0f7d)